pkg/controller/clone-controller.go

```diff
@@ -16,21 +16,26 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 
-	. "kubevirt.io/containerized-data-importer/pkg/common"
+	cdi "kubevirt.io/containerized-data-importer/pkg/common"
 	expectations "kubevirt.io/containerized-data-importer/pkg/expectations"
 )
 
 const (
-	// pvc annotations
+	//AnnCloneRequest sets our expected annotation for a CloneRequest
```

Comment:
It might be useful to define all annotations in _common_? That could make it easier to find the anno value, though IDE's pretty much mitigate that advantage...

Reply:
Yeah, agree for sure!  I'm just focused on doing the bare minimum necessary to make golint pass for now.  Then I have some changes that will do things like grouping and maybe rename things a bit.  Still kinda learning some of this code base so I'll look forward to help from you and others when I get to that point.